### PR TITLE
rewrite function

### PR DESCRIPTION
**Pull request by BBELAOUCHA**

### CODE REVIEW

**Comment by BBELAOUCHA** [comment]
Now all files follow pep8 style

File: MNNparcellation/bin/MNNparcellation [hunk 1]
```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
...
-# MNNparcellation -i input -o output -t tractogram -tb tractoprefix -seed coordinates
-# -Ex excluded -sm Similarity measures -n number of regions -cvth coeficient of
-# variance -nodif mask
+
+
+def run_parcellation(Arg):
+
+    if check_input(Arg):
+        print "Missing arguments check MNNparcellation --help"
+
```
Comment (gagdiez) [inline MNNparcellation/bin/MNNparcellation]: no need for new lines

File: MNNparcellation/bin/MNNparcellation [hunk 2]
```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
...
+                          default=0, help='''Enable (1) showing results in
+                          terminal''')
+
+    optional.add_argument('-m', action="store", dest='merge', type=int,
+                          default=0, help='Merge option: 0 (default)')
+
+    optional.add_argument('-Ex', action="store", dest='excluded',
+                          help='''file that contain seeds to be excluded from
+                          the parcellation''')
+    Arg = parser.parse_args()  # read different parameters
```
Comment (gagdiez) [inline MNNparcellation/bin/MNNparcellation]: name it args. Variable names should start with minuscule (PEP8)

File: MNNparcellation/bin/MNNparcellation [hunk 3]
```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
...
+    run_parcellation(Arg)
```
Comment (gagdiez) [inline MNNparcellation/bin/MNNparcellation]: # recomendation
This is ok, however, I'll recomend to change the signature of "run parcellation" in a way such you can call it as "run_parcellation(args.input, args.save, args.merge, args.excluded... etc)". This is exactly what I proposed in my python-talk. Check this script: https://github.com/AthenaEPI/logpar/blob/master/bin/cifti_parcellate

File: MNNparcellation/bin/MNNparcellation [hunk 4]
```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
 
 
 def check_input(Arg):
+
     test = False
-    if Arg.save == None or Arg.tractograms == None or Arg.tract_name == None or \
```
Comment (gagdiez) [inline MNNparcellation/bin/MNNparcellation]: When checking for None, use "is None".  if Arg.save is None or Arg.tractograms is None or Arg.tract_name is None or

File: MNNparcellation/bin/MNNparcellation [hunk 5]
```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
...
+        if Arg.NR:
+            Regions = [int(item) for item in Arg.NR.split(',')]
+
+        SM = ['Cosine']
+        # Default similarity measure, cosine similarity
+        if Arg.SM:
+            SM = [item for item in Arg.SM.split(',')]
+        # list conatining the wanted similarity measures
+        merge = 2
+        if Arg.merge is not None:
```
Comment (gagdiez) [inline MNNparcellation/bin/MNNparcellation]: if arg.merge: (you don't need the not None)

File: MNNparcellation/bin/MNNparcellation (hunk 2 above)
Comment (BBELAOUCHA) [reply MNNparcellation/bin/MNNparcellation]: token into account

**Pull request merged**